src/common/CMakeLists.txt

```diff
@@ -1,15 +1,14 @@
 add_subdirectory(binaries_op)
-add_subdirectory(bzip2_op)
 add_subdirectory(config)
 add_subdirectory(data_provider)
 add_subdirectory(dbsync)
 add_subdirectory(error_messages)
-add_subdirectory(file_op)
```

Comment:
is file_op no longer needed? can we remove the code?

Reply:
It is needed, but it is not needed in Windows. Efforts were made to completely remove it but several other libraries use it.

Reply:
I asked before I don't see any target linking against `file_op`'s target. And I don't see any other occurrence of `add_subdirectory(file_op)`. So I'm wondering how the target is being compiled, or if it's being used at all. Also, since it is not needed in windows, we could remove all the code in it enclosed with `#ifdef WIN32`.

`bzip2` has also been remove, they are probably not used by the currently added subdirectories, if not, then it's great to clean it up.

Reply:
Maybe when I tried to remove it was being used, and after several rebases it is no longer needed? I will try and let you know

Reply:
binaries_op is still using file_op, although you are wirght it is not explicity linked, it is used both the header and functions.